login_test.go

```diff
@@ -56,6 +57,7 @@ func createControler(t *testing.T) (*goa.Service, *LoginController) {
 }
 
 func TestTestUserTokenObtainedFromKeycloakOK(t *testing.T) {
+	os.Setenv("ALMIGHTY_DEVELOPER_MODE_ENABLED", "true")
```

Comment:
I promise I'm not trying to make this complex :) 

We should ideally set/unset the variable to it's original value in a `defer ....` .